### ISSUE TITLE
Cannot get satisfying results when training Faster RCNN on ImageNet dataset

### ISSUE THREAD

**Issue by Jaspereclipse** [rbgirshick/py-faster-rcnn#303]
I just finished training Faster R-CNN on ImageNet dataset (ILSVRC2013 Validation set, 100,000 iterations), see my [README.md](https://github.com/Jaspereclipse/py-faster-rcnn/blob/master/README.md) file for details. 

The training losses look like the following (parsed from log file with running average of every 100*20 = 2,000 iterations):
![loss_all_in_one_99980](https://cloud.githubusercontent.com/assets/9159309/17873942/40feb628-687d-11e6-8b0e-c39e9c3eae04.png)

When I use the final model to run the [demo](https://github.com/Jaspereclipse/py-faster-rcnn/blob/master/tools/demo_ilsvrc.py) code on a few pictures, the results are not quite what I expected:
![output_000456](https://cloud.githubusercontent.com/assets/9159309/17874098/5e5dab42-687e-11e6-8ee2-d97cf41b3492.jpg)
![output_001150](https://cloud.githubusercontent.com/assets/9159309/17874101/5e612a6a-687e-11e6-85a7-128c139e1bc7.jpg)
![output_001763](https://cloud.githubusercontent.com/assets/9159309/17874100/5e607bd8-687e-11e6-83ef-66dd9619c79f.jpg)
![output_004545](https://cloud.githubusercontent.com/assets/9159309/17874102/5e63a934-687e-11e6-818a-581ebf384801.jpg)
Many bboxes have a high score but are not any where close to the object.
I have no idea what went wrong. I will be grateful if there is someone who can help me with this issue.

**Comment by Jaspereclipse** [comment]
Closing this because it was caused by one stupid mistake I made.

**Comment by rtgoring** [comment]
Can you please describe the fix to your issue?
